### PR TITLE
Direction Indicator and Follow Solver Graduation

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Work to graduate the Direction indicator and Follow Solver into mainline

## Changes
- Fixes: #8998, #9006

### CODE REVIEW

**Comment by RogPodge** [comment]
DirectionalIndicatorGraduation

Indicator now scales with how much you have to turn to look at the target.

**Review by julenka — APPROVED** [review]
Looks great!

**Review by cre8ivepark — APPROVED** [review]
Thanks Roger! It looks great!

File: Assets/MRTK/SDK/Features/Utilities/Solvers/Solver.cs [hunk 1]
```diff
@@ -68,8 +68,8 @@ public float ScaleLerpTime
         }
 
         [SerializeField]
-        [Tooltip("If true, the Solver will respect the object's original scale values")]
-        private bool maintainScale = true;
+        [Tooltip("If true, the Solver will respect the object's original scale values on initialization")]
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/Utilities/Solvers/Solver.cs:71]: Should this have a `FormerlySerializedAs` tag?

Comment (RogPodge) [reply Assets/MRTK/SDK/Features/Utilities/Solvers/Solver.cs:71]: good catch, I'll make a followup PR to correct this

**Pull request merged**